juego.py

```diff
@@ -37,21 +37,47 @@ def __init__(self, level, fuente):
         self.expresion = fuente.render(self.primero + simbolo[operador] + self.segundo + " = ? ",True,(255,0,0))
         self.resultado = str(eval(self.primero+operacion[operador]+self.segundo))
         self.vida = 0
-                                           
+        
+        no_repeat_check_x = []
+        no_repeat_check_y = []
+        list_y = [int(sx(-50)),int(sx(-40)),int(sx(-30)),int(sx(-10)),int(sx(-20)),int(sx(-0)),int(sx(-60)),int(sx(-65))]
+        list_x = [int(sx(100)),int(sx(200)),int(sx(300)),int(sx(400)),int(sx(500)),int(sx(600)),int(sx(700)),
+                int(sx(800)),int(sx(900))]
+        def rand_generator_x():
+            count_x=0 
```

Comment:
Could you space the assignment operator from the variable name and value, helps with readability.